package.json

```
@@ -89,6 +89,11 @@
                     "command": "zeplin.sidebar.unpinAll",
                     "when": "view == zeplin.views.pinned && zeplin:sidebar:anyPinnedItems",
                     "group": "zeplin.sidebar.pinned_2"
+                },
+                {
+                    "command": "zeplin.sidebar.refresh",
+                    "when": "view == zeplin.views.activity",
```

Is it possible to merge this with the above command like this?
```suggestion
                    "when": "view == zeplin.views.barrels || view == zeplin.views.activity",
```

---

I thought I wouldn't work but I tried it after you said and it works. 🎉 A patch is coming.